### PR TITLE
fix(chai-dom-equals): ignore style tag in shadow dom by default

### PR DESCRIPTION
**Pull request by daKmoR**

### CODE REVIEW

File: packages/chai-dom-equals/chai-dom-equals.js [hunk 1]
```diff
@@ -100,9 +100,9 @@ export const chaiDomEquals = (chai, utils) => {
   const domEquals = _super => function handleDom(value, ...args) {
     // @ts-ignore
     if (utils.flag(this, 'shadowDom')) {
-      const expectedHTML = getDiffableHTML(value, args[0]);
+      const expectedHTML = getDiffableHTML(value, { ignoreTags: ['style'], ...args[0] });
```
Comment (LarsDenBakker) [inline packages/chai-dom-equals/chai-dom-equals.js]: this does not merge user-provided `ignoreTags`

Comment (daKmoR) [reply packages/chai-dom-equals/chai-dom-equals.js]: yeah it overrides... which would be good I would say...
```js
expect(el).shadowDom.to.equal('...'); // ignores style tag
expect(el).shadowDom.to.equal('...', { ignoreTags: ['foo'] }); // does not ignores style tag
// e.g. you will need to ignore it yourself if you want to keep ignoring
expect(el).shadowDom.to.equal('...', { ignoreTags: ['foo', 'style'] });
```

Comment (LarsDenBakker) [reply packages/chai-dom-equals/chai-dom-equals.js]: I think that's a confusing API, and it's an extra thing to explain/document. Maybe the better approach is to just ignore style, svg and script tags entirely and not just the content. Directly in `semantic-dom-diff` (basically the same thing we had before). That way at least it's straightforward

Comment (daKmoR) [reply packages/chai-dom-equals/chai-dom-equals.js]: true let's ignore it globally for now... if there is a need to override these default we can look at it when the need arises...

**Pull request closed without merging**